flink-core/src/main/java/org/apache/flink/util/NetUtils.java

```diff
@@ -309,6 +313,19 @@ public static ServerSocket createSocketFromPorts(Iterator<Integer> portsIterator
 		return null;
 	}
 
+	/**
+	 * Returns the wildcard address to listen on all interfaces.
+	 * @return Either 0.0.0.0 or :: depending on the IP setup.
+	 */
+	public static String getWildcardIPAddress() {
```

Comment:
I chose not to do that to avoid throwing exceptions in a static initializer. The current implementation preserves the context of the exception. I'm ok with initializing it statically.